File: android/mobilepasssdk/src/main/java/com/armongate/mobilepasssdk/model/response/ResponseAccessPointListItem.java
package com.armongate.mobilepasssdk.model.response;

import androidx.annotation.Nullable;

public class ResponseAccessPointListItem {
    /** Id */
    public @Nullable String i;
    /** Name */
    public @Nullable String n;
    /** QR Codes */
    public @Nullable ResponseAccessPointListQRCode[]    q;
    /** Terminals */
    public @Nullable ResponseAccessPointListTerminal[]  t;
    /** GeoLocation */
    public @Nullable ResponseAccessPointListGeoLocation g;

    /** QR Code Ids | Only for mapping on storage */
    public String[] d;
}

File: ios/MobilePassSDK/Model/Response/ResponseAccessPointListItem.swift
//
//  ResponseAccessPointListV2Item.swift
//  MobilePassSDK
//
//  Created by Erinc Cakir on 15.08.2021.
//

import Foundation

struct ResponseAccessPointListItem: Codable {
    /** Id */
    var i: String?
    /** Name */
    var n: String?
    /** QR Codes */
    var q: [ResponseAccessPointListQRCode]?
    /** Terminals */
    var t: [ResponseAccessPointListTerminal]?
    /** GeoLocation */
    var g: ResponseAccessPointListGeoLocation?
    
    /** QR Code Ids | Only for mapping on storage */
    var d: [String]?
}
